packages/ui-api/src/with/call.tsx

```diff
@@ -166,18 +166,16 @@ export default function withCall<P extends ApiProps> (
             fn,
             params,
             true,
-            false
+            ''
           ];
         }
 
-        const endpoints = [endpoint].concat(fallbacks || []);
+        const endpoints: string[] = [endpoint].concat(fallbacks || []);
         let apiSection, area, section, method;
-        let i = 0;
 
-        while (!apiSection && i < endpoints.length) {
-          [endpoint, apiSection, area, section, method] = this.constructApiSection(endpoints[i]);
-          i++;
-        }
+        [endpoint, apiSection, area, section, method] = endpoints
```

Comment:
Could remove 176 and just `const [...]` here?

Reply:
I'm overwriting the passed endpoint here (https://github.com/polkadot-js/apps/pull/1416/files#diff-e2040f6b00b230fed0e435eb6386b9a3R162), so const doesn't work because it's declared already.

Reply:
Ahh yes, cool. But I also don't see us actually it, i.e. the `endpoint` as overwritten downwards? So if could even be as simple as `const [, apiSection, area, section, method] = endpoints`

(Or even not returning it, since it is not actually needed atm)